node-graph/gstd/src/raster.rs

```diff
@@ -3,10 +3,10 @@ use fastnoise_lite;
 use glam::{DAffine2, DVec2, Vec2};
 use graphene_core::instances::Instance;
 use graphene_core::raster::bbox::Bbox;
-use graphene_core::raster::image::{Image, RasterDataTable};
-use graphene_core::raster::{Alpha, AlphaMut, Bitmap, BitmapMut, CellularDistanceFunction, CellularReturnType, Channel, DomainWarpType, FractalType, LinearChannel, Luminance, NoiseType, RGBMut};
+use graphene_core::raster::image::RasterDataTable;
+pub use graphene_core::raster::*;
```

Comment:
@TrueDoctor that's the reexport I added. But since this mod already contains a `mod image` through the node macro, that mod takes priority over the glob reexport. And I can't insert a glob reexport into the image mod itself, since it's fully contained within the node macro's output.